RadarSDK/Radar.m

```diff
@@ -215,6 +215,33 @@ + (void)ipGeocodeWithCompletionHandler:(RadarIPGeocodeCompletionHandler)completi
     }];
 }
 
++ (void)getContextWithCompletionHandler:(RadarGetContextCompletionHandler)completionHandler {
+    [[RadarLocationManager sharedInstance] getLocationWithCompletionHandler:^(RadarStatus status, CLLocation * _Nullable location, BOOL stopped) {
+        if (status != RadarStatusSuccess) {
+            if (completionHandler) {
+                completionHandler(status, nil, nil);
+            }
+            
+            return;
```

Comment:
not sure if we actually need a `return` here? the convention is a bit sporadic in the rest of the codebase, might be an opportunity to standardize how we handle error / empty state callbacks